query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/schema/RelationOptions.java

```diff
@@ -97,22 +111,32 @@ default SelfT withCompaction(@NonNull CompactionStrategy<?> compactionStrategy)
   }
 
   /**
-   * Configures compression using the LZ4 algorithm with the given chunk length and crc check
-   * chance.
-   *
-   * @see #withCompression(String, int, double)
+   * @deprecated This method only exists for backward compatibility. Use {@link
+   *     #withLZ4Compression(int)} instead.
    */
+  @Deprecated
```

Comment:
:+1: seems reasonable to me as it maintains backwards compatibility with older versions, should also note that this will not work with C* 5.0+